### PR TITLE
Add maxiang.app Latest

### PR DESCRIPTION
**Pull request by filosfino**
### Checklist
- [x] The commit message includes the cask’s name and version.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] Checked there are no open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there are no closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/maxiang.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask 'maxiang' do
+  version ':latest'
```
Comment (vitorgalvao) [inline Casks/maxiang.rb]: `version :latest` (no quotes).

File: Casks/maxiang.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  sha256 'ba9f2f32bc6c68ac732d0131d1e2a74aae2125a44012a0f6c1e975f7a4676419'
```
Comment (vitorgalvao) [inline Casks/maxiang.rb]: `sha256 :no_check` (always accompanies `version :latest`).

File: Casks/maxiang.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+  # github.com/filosfino was verified as official when first introduced to the cask
+  url 'https://github.com/filosfino/apps/raw/master/Maxiang.dmg'
```
Comment (vitorgalvao) [inline Casks/maxiang.rb]: Can you point me to this link on the homepage?

File: Casks/maxiang.rb [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+  name 'maxiang'
```
Comment (vitorgalvao) [inline Casks/maxiang.rb]: Also needs the original name. See [docs](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/name.md).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/maxiang.rb (hunk 3 above)
Comment (filosfino) [inline Casks/maxiang.rb]: u mean the repo url on github? that is https://github.com/filosfino/apps

File: Casks/maxiang.rb (hunk 4 above)
Comment (filosfino) [inline Casks/maxiang.rb]: Does it has to be in utf-8? Can chinese characters be in here?

Comment (vitorgalvao) [inline Casks/maxiang.rb]: Yes, just like you have in `app`. Make it one “english” and one original.

File: Casks/maxiang.rb (hunk 3 above)
Comment (vitorgalvao) [inline Casks/maxiang.rb]: I still need this, to verify the link

Comment (filosfino) [inline Casks/maxiang.rb]: > Can you point me to this link on the homepage?

What do you mean by that?

File: Casks/maxiang.rb (hunk 4 above)
Comment (filosfino) [inline Casks/maxiang.rb]: How about this?
```
  name '马克飞象'
  name 'marxico'
  name 'maxiang'
```

File: Casks/maxiang.rb (hunk 3 above)
Comment (filosfino) [inline Casks/maxiang.rb]: If you mean the repo home page, that is:
https://github.com/filosfino/apps

Comment (vitorgalvao) [inline Casks/maxiang.rb]: No, I mean that from the homepage I can’t see any proof this download link or that repo are in any way official. I need some detail on the page that makes it clear.

File: Casks/maxiang.rb (hunk 4 above)
Comment (vitorgalvao) [inline Casks/maxiang.rb]: Make it just

```ruby
name 'maxiang'
name '马克飞象'
```

File: Casks/maxiang.rb (hunk 3 above)
Comment (filosfino) [inline Casks/maxiang.rb]: I see. Here's the senario, I'm just a user of the app.
Guess what I can do is asking the author to provide an official link?

File: Casks/maxiang.rb [hunk 5]
```diff
@@ -0,0 +1,14 @@
+cask 'maxiang' do
+  version :latest
+  sha256 :no_check
+
+  # github.com/filosfino was verified as official when first introduced to the cask
+  url 'https://github.com/filosfino/apps/raw/master/Maxiang.dmg'
+  name 'ma ke fei xiang'
+  name 'marxico'
+  name 'maxiang'
+  homepage 'https://github.com/filosfino/apps/raw/master/Maxiang.dmg'
```
Comment (vitorgalvao) [inline Casks/maxiang.rb]: This is not a homepage.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/maxiang.rb (hunk 3 above)
Comment (vitorgalvao) [inline Casks/maxiang.rb]: We can’t accept non-official links. It’s a security risk for users, and they need to trust the source. We need the homepage to provide and official link.

**Pull request closed without merging**